DangerSwap/DangerSwap/Utilities/NationalitySelector.cs

```diff
@@ -3,6 +3,7 @@
     using System.Globalization;
     public static class NationalitySelector
     {
+        //Mindaugas
```

Comment:
Line 9, C# provides us syntax sugar, which automatically specifies the type. That's why I'd suggest you to use `var` keyword instead of the real type.